poppy/poppy_core.py

```diff
@@ -2273,8 +2273,10 @@ def get_phasor(self,wave):
                 lx,ly=resampled_amplitude.shape
                 #crop down to match size of wavefront:
                 lx_w,ly_w = wave.amplitude.shape
-                border_x = np.abs(np.floor((lx-lx_w)/2))
-                border_y = np.abs(np.floor((ly-ly_w)/2))
+
+                # explicit cast to int fixes index errors in Numpy >= 1.12.0
+                border_x = int(np.abs(np.floor((lx-lx_w)/2)))
```

Comment:
I'm indifferent about the comments.

However, for this line, there are three function calls where there should be a single operator:

```
In [12]: lx_w, lx = 20, 45

In [13]: int(np.abs(np.floor((lx-lx_w)/2)))
Out[13]: 12

In [14]: (lx - lx_w) // 2
Out[14]: 12
```

Since the `shape` elements are integers, integer division is the way to go.